docs/userguide.md

```diff
@@ -82,98 +82,7 @@ Also, please make sure the joining cluster name(s) provided matches the joining
 cluster context from your kubeconfig. This will already be the case if you used
 the minikube instructions above to create your clusters.
 
-## Manual Deployment
-
-If you'd like to understand what the script is automating for you, then proceed
-by manually running the commands below.
-
-### Deploy the Cluster Registry CRD
-
-First you'll need to create the reserved namespace for registering clusters
-with the cluster registry:
-
-```bash
-kubectl create ns kube-multicluster-public
-```
-
-Using this repo's vendored version of the [Cluster
-Registry](https://github.com/kubernetes/cluster-registry), run the
-following:
-
-```bash
-kubectl apply --validate=false -f vendor/k8s.io/cluster-registry/cluster-registry-crd.yaml
-```
-
-### Deploy Federation
-
-First you'll need to create a permissive rolebinding to allow federation
-controllers to run. This will eventually be made more restrictive, but for now
-run:
-
-```bash
-kubectl create clusterrolebinding federation-admin --clusterrole=cluster-admin \
-    --serviceaccount="federation-system:default"
-```
-
-Now you're ready to deploy federation v2 using the existing YAML config. This
-config creates the `federation-system` namespace, RBAC resources, all the CRDs
-supported, along with the service and statefulset for the
-federation-controller-manager.
-
-```bash
-kubectl -n federation-system apply --validate=false -f hack/install-latest.yaml
-```
-
-**NOTE:** The validation fails for harmless reasons on kube >= 1.11 so ignore validation
-until `kubebuilder` generation can pass validation.
-
-Verify that the deployment succeeded and is available to serve its API by
-seeing if we can retrieve one of its API resources:
-
-```bash
-kubectl -n federation-system get federatedcluster
-```
-
-It should successfully report that no resources are found.
-
-### Enabling Push Propagation
```

Comment:
(No action required) I will be updating the doc to include mention of `kubefed2 federate enable` which will replace this section.